.github/workflows/github-actions-demo.yml

```diff
@@ -0,0 +1,16 @@
+name: GitHub Actions Demo
```

Comment:
Name of the pipeline and name of this file could be changed to reflect what this workflow actually does